mq/publishers.go

```diff
@@ -249,3 +250,48 @@ func sendPeers() {
 		}
 	}
 }
+
+func sendDNSSync() error {
```

Comment:
this func should be simplified to all get all entries from the DB and push it to clients

Reply:
Removed the cache in new commit. Please help review again.